fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java

```diff
@@ -59,6 +63,8 @@
 
     private static final Pattern TRAILING_SLASH_REGEX = Pattern.compile("/+$");
 
+    private static final Pattern MEMENTO_PATH_PATTERN = Pattern.compile(".*/fcr:versions/(.*)$");
```

Comment:
Yes, to using the constant, but no to `\\d`. We actually don't want the more specific version here, because we have logic for returning helpful error messages for malformed mementos. I want it to match anything with `fcr:versions` regardless of if it's correct, so that I can throw an error if I can't interpret it as a memento. I'll leave a comment so that it's clear this is intended, though integration tests would break if it was changed.